adal/src/androidTest/java/com/microsoft/aad/adal/ReflectionUtils.java

```diff
@@ -28,9 +28,13 @@
 import java.lang.reflect.InvocationTargetException;
 import java.lang.reflect.Method;
 
-public class ReflectionUtils {
+public final class ReflectionUtils {
 
-    public final static String TEST_PACKAGE_NAME = "com.microsoft.aad.adal";
+    public static final String TEST_PACKAGE_NAME = "com.microsoft.aad.adal";
+    
+    private ReflectionUtils() {
+        //not called
```

Comment:
Are we adding this because of PMD warning? If so, @addev-ashish as we talked we need to suppress it. I don't see a reason or benefit of adding this obvious comment in every empty private constructor.